src/qt/uicomponents/MeterPanelHandler.cpp

```diff
@@ -13,15 +13,7 @@ qreal MeterPanelHandler::Value() const
 
 void MeterPanelHandler::SetValue(qreal value)
 {
-   if (value < 0.0) {
-      qDebug() << "MeterPanelHandler value cannot be less than 0";
-      return;
-   }
-
-   if (value > 1.0) {
-      qDebug() << "MeterPanelHandler value cannot be greater than 1";
-      return;
-   }
+   Q_ASSERT(value >= 0.0 && value <= 1.0);
```

Comment:
1. I would prefer the standard `assert`
2. Asserting should not replace the checks

Reply:
Done.